src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -1,7 +1,7 @@
 (ns puppetlabs.services.ca.certificate-authority-core
   (:import  [java.io InputStream])
   (:require [puppetlabs.puppetserver.certificate-authority :as ca]
-            [puppetlabs.certificate-authority.core :as ca-utils]
+            [puppetlabs.ssl-utils.core :as utils]
```

Comment:
So, this would break the usages of `ca-utils` since those weren't renamed to `ssl-utils`...but there aren't any!! We can just delete this :require altogether